Introduce combobox-committed event (#18)
github/combobox-nav: README.md

## Events

> - `combobox-commit` (bubbles) - fired on the `role="option"` element when it is activated.

Reviewer: Should this be cancelable to justify the two events?

> - `combobox-committed` (bubbles) - fired on the `role="option"` element on the next tick of the activation event. This is useful for when the activation comes with side-effects like state changes that need to inspect afterward the fact.

Reviewer: ```suggestion
- `combobox-committed` (bubbles) - fired on the `role="option"` element on the next tick of the event loop. This is useful for when the activation comes with side-effects, like a labeled input's `checked` state that needs to be inspected.
```

```js
list.addEventListener('click', function(event) {

>   noneSelectedMessage.hidden = true

Reviewer: We should use the label input example since that's what motivates this change.
